automation/e2e-source.sh

```diff
@@ -3,3 +3,6 @@
 export DEV_MODE="${DEV_MODE:-false}"
 export STORAGE_CLASS="${STORAGE_CLASS:-}"
 export NUM_NODES=${NUM_NODES:-2}
+export IN_CLUSTER_IMAGE_REGISTRY="image-registry.openshift-image-registry.svc:5000"
+export IN_CLUSTER_REGISTRY_ACCESS_KEY_ID=""
+export IN_CLUSTER_REGISTRY_SECRET_KEY=""
```

Comment:
@0xFelix @ksimon1 It seems like OCP's in-cluster registry requires authorization (returns 401 if username/password not set).

Reply:
have you tried https://access.redhat.com/solutions/6959306?